tickets_plus/cogs/override.py

```diff
@@ -80,14 +80,17 @@ async def reload(self,
             ctx: The interaction context.
             sync: Whether to sync the tree after reloading cogs.
         """
-        await ctx.response.send_message("Reloading cogs...")
+        await ctx.response.defer()
```

Comment:
No point deferring here if you're going to send something immediately afterwards anyway?